app/views/enterprises/_info.html.erb

```diff
@@ -1,28 +1,114 @@
-<div class="notification-box upsale-notification">
-  <div class="notification-box--content">
-    <h3><%= t('admin.enterprise.upgrade_to_ee') %></h3>
-    <%= image_tag "enterprise_edition.png", class: "widget-box--teaser-image" %>
-
-    <p><%= t('homescreen.blocks.upsale.description') %></p>
-
-    <ul class="">
-      <li>
-        <%= t('homescreen.blocks.upsale.additional_features') %>
-      </li>
-      <li>
-        <%= t('homescreen.blocks.upsale.professional_support') %>
-      </li>
-    </ul>
-    <p>
-      <b><%= t('homescreen.blocks.upsale.become_hero') %></b> <%= t('homescreen.blocks.upsale.you_contribute') %>
-    </p>
-    <%= link_to( "#{OpenProject::Static::Links.links[:upsale][:href]}/?utm_source=unknown&utm_medium=community-edition&utm_campaign=enterprise-admin",
-                 { class: 'button -alt-highlight',
-                   target: '_blank',
-                   aria: {label: t('admin.enterprise.order')},
-                   title: t('admin.enterprise.order')}) do %>
-      <%= op_icon('button--icon icon-add') %>
-      <span class="button--text"><%= t('admin.enterprise.order') %></span>
-    <% end %>
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) 2012-2020 the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2017 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See docs/COPYRIGHT.rdoc for more details.
+
+++#%>
+<% content_for :header_tags do %>
```

Comment:
@ingamai I found the problem. This doesn't work in partials (_info.html.erb). If you move this bit to show.html.erb t should work!